src/components/add-transaction-dialog-select/add-transaction-dialog-select.js

```diff
@@ -0,0 +1,57 @@
+import template from "./add-transaction-dialog-select.html";
+
+import { MDCSelect } from "@material/select";
+
+export class AddTransactionSelectComponent {
+  constructor(mountPoint, props) {
+    this.mountPoint = mountPoint;
+    this.props = props;
+  }
+
+  clean() {
+    this.select.selectedIndex = -1;
+  }
+
+  getTag() {
+    return this.getValue();
+  }
+
+  getAccount() {
+    return this.props.items.find(item => {
+      return item.name == this.getValue();
+    });
+  }
+
+  makeList() {
+    return this.props.items.map(item => (item.name ? item.name : item));
```

Comment:
what does this do?

Reply:
there are two types of stored data for selects. 
- tags have structure ["tag1", "tag2", .... ]
- accounts have structure [{name: "account1", type:"type1", currency: "curr1"}, ...]

I need only names of both.

I will change it to another implementation:
if (tags) return tags
else return array of account names